movie_search/media_api.py

```diff
@@ -87,17 +85,15 @@ def get_media_data(
     if query is not None:
         params.update({"query": query})
 
-    print("Params: ", params)
-
     response = requests.get(url, params=params)
 
     return response.json()
 
 def main():
     from utils import dump_movie_data_to_json
-    data = get_media_data("/watch/providers/tv")
+    data = get_media_data("/movie/153/recommendations") 
```

Comment:
should this 153 be hardcoded?

Reply:
That was only for testing when running media_api.py directly. I run media_api.py directly when I want to inspect the JSON data more.